### ISSUE TITLE
Hard-coded maximum size of Chat Component

### ISSUE THREAD

**Issue by emlynmac** [microsoft/fluentui-contrib#169]
From the code here: 
https://github.com/microsoft/fluentui-contrib/blob/170e83e3c102192b227d1a7f06642bd39fcf953f/packages/react-chat/src/components/Chat/useChatStyles.styles.ts#L26

The `maxWidth` of the `Chat` component is hardcoded to 1056px. 

This is causing issues with a 1P customer, and I'd like to understand where this value originates from.

Could this be removed ?

**Comment by YuanboXue-Amber** [comment]
Hi @emlynmac, all existing styles can be overridden. You can add `maxWidth: 'unset'` to Chat: 
https://stackblitz.com/edit/v7ers1?file=src%2Fexample.tsx

**Comment by emlynmac** [comment]
> Hi @emlynmac, all existing styles can be overridden. You can add `maxWidth: 'unset'` to Chat: https://stackblitz.com/edit/v7ers1?file=src%2Fexample.tsx

Yes, and we have; the question is why is this limit in the code to start with?
